base/applications/shutdown/lang/pt-PT.rc

```diff
@@ -0,0 +1,103 @@
+/* Translation and UTF-8 Conversion by Gabriel Aguiar(fgsoftware1) <fgygh5804@gmail.com,fgsoftwarestudio@gmail.com> (July, 2021) */
+
+LANGUAGE LANG_PORTUGUESE, SUBLANG_NEUTRAL
+
+IDD_GUI DIALOGEX 0, 0, 240, 255
+STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU
+CAPTION "Remote Shutdown"
+FONT 8, "MS Shell Dlg"
+BEGIN
+    DEFPUSHBUTTON "&OK", IDC_OK, 125, 232, 50, 14
+    PUSHBUTTON "&Cancelar", IDC_CANCEL, 178, 232, 50, 14
+    LTEXT "Co&mputadores:", IDC_STATIC, 9, 9, 35, 36
+    LISTBOX IDC_COMPUTER_LIST, 8, 19, 162, 55
+    PUSHBUTTON "&Adicionar...", IDC_ADD_SYSTEM, 179, 19, 50, 14
+    PUSHBUTTON "&Remover", IDC_REMOVE_SYSTEM, 179, 36, 50, 14, WS_DISABLED
+    PUSHBUTTON "&Navegar...", IDC_BROWSE_SYSTEM, 179, 53, 50, 14
+    LTEXT "Ação", IDC_ACTION, 11, 81, 20, 14
+    COMBOBOX IDC_ACTION_TYPE, 37, 79, 129, 14, WS_TABSTOP | CBS_DROPDOWN
+    CHECKBOX "Avisar utilizadores", IDC_WARN_USERS, 175, 79, 55, 14, BS_AUTOCHECKBOX | WS_TABSTOP
+    LTEXT "Mostrar aviso para", IDC_SHOW_WARN_ONE, 11, 99, 65, 14
+    EDITTEXT IDC_SHOW_WARN, 78, 97, 41, 14
+    LTEXT "segundo(s)", IDC_SHOW_WARN_TWO, 124, 99, 32, 10
+    GROUPBOX "A encerrar o rastreador de eventos", IDC_STATIC, 5, 114, 224, 114
+    LTEXT "Motivo:", IDC_STATIC, 16, 130, 27, 8
+    CHECKBOX "Planeado", IDC_PLANNED, 175, 130, 40, 12, BS_AUTOCHECKBOX | WS_TABSTOP
+    COMBOBOX IDC_REASON_CODE, 17, 142, 198, 13, WS_TABSTOP | CBS_DROPDOWN
+    LTEXT "Come&ntar:", IDC_COMMENT_CAPTION, 16, 159, 38, 8
+    EDITTEXT IDC_COMMENT_TEXT, 17, 171, 198, 50, WS_VSCROLL
+END
+
+/* Information and error messages */
+STRINGTABLE
+BEGIN
+    IDS_USAGE "Utilitário de encerramento do ReactOS\n\
+\n\
+Usage: shutdown [/?] [/i | /l | /s | /r | /g | /a | /p | /h | /e] [/f]\n\
+       [/m \\\\computer][/t xxx][/d [p|u:]xx:yy [/c ""comment""]]\n\
+\n\
+    No arguments or /?    Display this help.\n\
+    /i      Mostrar a interface de utilizador(UI). This option must be the\n\
+            a primeira.\n\
+    /l      Terminar sessão apenas no sistema local. Nao pode ser usado com /m or /d.\n\
+    /s      Encerramento do computador.\n\
+    /r      Reinicio do computador.\n\
+    /g      Reinicio do computador e todas as aplicações registadas.\n\
+    /a      Cancelar um encerramento atrasado. Só pode ser usado durante o periodo de\n\
+            atraso.\n\
+    /p      A encerrar computador local sem nenhum aviso de exceções. Pode ser\n\
```

Comment:
'exepção' not 'exeção' This 'p' is very audible 
https://pt.wikipedia.org/wiki/Acordo_Ortogr%C3%A1fico_de_1990